index.html

```diff
@@ -139,10 +139,10 @@ <h3>Simple Budgeting</h3>
   <section class="partner">
     <h3>Partner</h3> <hr>
     <ul>
-      <li>Bank-Central</li>
-      <li>Guinea-Games</li>
-      <li>freeCodeCamp</li>
+      <li>Google</li>
+      <li>Central-Banking</li>
       <li>Microverse</li>
+      <li>freeCodeCamp</li>
```

Comment:
- [ OPTIONAL ] I think it would be great if you make the bg color for the footer in your home page in white background color since it like this in the given design but I am assiging as optional because the previous reviewer didn't mention it but please change it 🙏🏻🤩✨💪🏻 


### Kindly remove the gap shown between the partner section and the footer ✨💪🏻 and regarding to social media links I think also you can remove it since it is not exist in the given design 🙏🏻💯

- Your design
![image](https://github.com/aradradev/easyBankTC/assets/58553711/12e590e8-b040-447b-b658-11f6eb0100b3)


- Given
![image](https://github.com/aradradev/easyBankTC/assets/58553711/cc342b07-3383-45ea-ba6c-367277dfceaa)